eth-contracts/contracts/service/MockServiceProviderFactory.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.5.0;
+
+import "./registry/RegistryContract.sol";
+
+
+// Test contract used in claim factory scenarios
```

Comment:
v1 - Move this to a test directory or something, this should not be around the main contracts